graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/builtins/ArrayPrototypeBuiltins.java

```diff
@@ -366,6 +369,9 @@ protected Object createNode(JSContext context, JSBuiltin builtin, boolean constr
                 return JSArrayGroupByNodeGen.create(context, builtin, args().withThis().fixedArgs(2).createArgumentNodes(context));
             case groupByToMap:
                 return JSArrayGroupByToMapNodeGen.create(context, builtin, args().withThis().fixedArgs(2).createArgumentNodes(context));
+
+            case toReversed:
+                return ArrayPrototypeBuiltinsFactory.JSArrayToReversedNodeGen.create(context, builtin, false, args().withThis().fixedArgs(0).createArgumentNodes(context));
```

Comment:
If you add something very similar to those two lines (and the one entry above) also to the TypedArrayPrototypeBuiltins.java, then this should also work there.